packages/nimble-components/src/theme-provider/index.ts

```diff
@@ -33,19 +49,50 @@ export const theme = DesignToken.create<Theme>({
  * @internal
  */
 export class ThemeProvider extends FoundationElement {
-    @attr({
-        attribute: 'direction'
-    })
+    @attr()
+    public override lang = '';
+
+    @attr()
     public direction: Direction = Direction.ltr;
 
-    @attr({
-        attribute: 'theme'
-    })
+    @attr()
     public theme: Theme = Theme.light;
 
+    public get validity(): ValidityObject {
+        return {
+            invalidLang: this.langIsInvalid
+        };
+    }
+
+    private langIsInvalid = false;
+
+    public checkValidity(): boolean {
```

Comment:
Should we use the `Validator` base class here so that we aren't manually tracking the validity keys in this class?

Reply:
It's so trivial that I think it's actually simpler this way. If we end up having to do more validation, I would probably start using a `Validator`.